<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>486</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>424</width>
    <height>486</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PolyCalc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_input">
      <property name="title">
       <string>Enter coefficients</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_real">
             <property name="text">
              <string>Real part</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_imag">
             <property name="text">
              <string>Imaginary part</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_a">
           <property name="text">
            <string>Coefficient a:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_ar">
             <property name="minimum">
              <double>-999999999999.989990234375000</double>
             </property>
             <property name="maximum">
              <double>999999999999.989990234375000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_ai">
             <property name="minimum">
              <double>-999999999999.989990234375000</double>
             </property>
             <property name="maximum">
              <double>999999999999.989990234375000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_b">
           <property name="text">
            <string>Coefficient b:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_br">
             <property name="minimum">
              <double>-999999999999.989990234375000</double>
             </property>
             <property name="maximum">
              <double>999999999999.989990234375000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_bi">
             <property name="minimum">
              <double>-999999999999.989990234375000</double>
             </property>
             <property name="maximum">
              <double>999999999999.989990234375000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_c">
           <property name="text">
            <string>Coefficient c:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_cr">
             <property name="minimum">
              <double>-999999999999.989990234375000</double>
             </property>
             <property name="maximum">
              <double>999999999999.989990234375000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_ci">
             <property name="minimum">
              <double>-999999999999.989990234375000</double>
             </property>
             <property name="maximum">
              <double>999999999999.989990234375000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Number part:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_arg">
           <property name="text">
            <string>Argument x:</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_argr">
             <property name="minimum">
              <double>-999999999999.989990234375000</double>
             </property>
             <property name="maximum">
              <double>999999999999.989990234375000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_argi"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="calculateButton">
         <property name="text">
          <string>Calculate</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_output">
      <property name="title">
       <string>Results</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPlainTextEdit" name="resultTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Calculation results will appear here.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
